### PR TITLE
Counter App (TASK-1)-Jhanvi Madan

### PR DESCRIPTION
**Pull request by Jhanvimadan**

### CODE REVIEW

File: Counter App (Assignment)-Jhanvi Madan
Comment (i-am-SnehaChauhan) [inline Counter App (Assignment)-Jhanvi Madan:1]: Please you provide screenshots or a video of the page.

**Review by i-am-SnehaChauhan — CHANGES_REQUESTED** [review]
Screenshot or video demonstration not provided

File: Counter App (Assignment)-Jhanvi Madan
Comment (Jhanvimadan) [reply Counter App (Assignment)-Jhanvi Madan:1]: https://github.com/am-igdtuw/MENTORSHIP-COHORT1.0/assets/124586881/fa108797-cb8d-4487-9dae-d823a3b92dce

Comment (Jhanvimadan) [reply Counter App (Assignment)-Jhanvi Madan:1]: Earlier my video wasn't uploading so I put the link in the code itself as comment.

**Pull request closed without merging**